### DISCUSSION TITLE
Unable to detect private go modules inside subgroup on custom GitLab EE host

### DISCUSSION THREAD

**Discussion by lucas-koontz** [renovatebot/renovate#10294] [Request Help] [answered]
I'm getting the following error for some of my private **Go** modules hosted in Gitlab.

This is the relevant information I have:

`Failed to look up dependency privatehost.com/group/subgroup/goModuleA`

It seems not to be an authentication issue because another private module not in a subgroup can be found (`privatehost.com/group/goModuleB`)

**Comment by HonkingGoose** [comment, marked as answer]
Renovate bot does not (yet) support private go modules on custom GitLab EE hosts.

You can follow issue https://github.com/renovatebot/renovate/issues/4601 for the feature request to "support private go modules on custom GitLab EE hosts".
